cfme/utils/ext_auth.py

```diff
@@ -56,8 +55,11 @@ def setup_external_auth_ipa(**data):
 
         ensure_browser_open()
         appliance.current_appliance.server.login_admin()
-        if data["ipaserver"] not in get_ntp_servers():
-            set_ntp_servers(data["ipaserver"])
+
+        if data["ipaserver"] not in (
+                appliance.current_appliance.server.settings.ntp_servers_form.values()):
```

Comment:
**REQUIIRED:** Wait what?! ```appliance.current_appliance```? Oh wait I see......appliance module is imported. As you're in here, please can you clean this up, we have ```current_appliance``` set on line 39/38 then we use ```appliance.current_appliance```.

Reply:
Done